### PR TITLE
Moves scripts to ./bin and /usr/local/bin

### LINKED ISSUE #21: Path Confusion
In the repo all of the commands are located at `./script/` however in the container they're located at `/bin/script`. Also the other tool that is being installed in the image at `/usr/local/bin`. Perhaps if everything was located at `/usr/local/bin` it would be a little more conventional.

### PR DESCRIPTION
**Pull request by johnallen3d**
Resolves #21

### CODE REVIEW

**Comment by blankenshipz** [comment]
@johnallen3d WIP? I see more 🔴  then 💚

**Comment by blankenshipz** [comment]
e.g `git add ./bin`?

**Comment by johnallen3d** [comment]
Whoops!

File: bin/bootstrap [hunk 1]
```diff
@@ -6,8 +6,8 @@
 
 set -e
 
-DIR=/bin/script
-source "$DIR/common"
+DIR=/usr/local/bin
```
Comment (blankenshipz) [inline bin/bootstrap]: Perhaps these lines should be simplified to `source "/usr/local/bin/_common"` for cohesion?

**Comment by blankenshipz** [comment]
LGTM 👏 :shipit:

**Pull request merged**